utils/db.go

```diff
@@ -38,12 +40,43 @@ func openSQL(user, password, database string) (*sql.DB, error) {
 	}
 }
 
-// Convert is an internal function for Copy methods
-func Convert(redisType, sqluser, sqlpassword, sqldatabase, sqltable, redisaddr, redispass string, log bool) error {
-	db, err := openSQL(sqluser, sqlpassword, sqldatabase)
+// openPostgres opens a PostgreSQL connection with a desired user, password database name, host and port
+func openPostgres(user, password, database, host, port string) (*sql.DB, error) {
+	connectionString := fmt.Sprintf("postgres://%s:%s@%s:%s/%s?sslmode=disable", user, password, host, port, database)
```

Comment:
`sslmode=prefer` would allow somebody to for example use this on a local docker box without SSL, this for example would not work on any hosted DB (think AWS RDS, CosmosDB etc)

Reply:
ssl mode changed to "prefer"